### PR TITLE
Filterable List: Allow Post date description field to be blank

### PR DESCRIPTION
**Pull request by schbetsy**
## Changes

- Give the "Post date description" field a more descriptive label and help text
- Remove the default value and make it non-required, so it may be left blank.

## Testing

1. Pull down this branch, get a recent database dump, run all migrations
2. Visit a filterable list page, such as http://localhost:8000/policy-compliance/enforcement/actions/
    - notice that each post in the filterable list says "Date filed" before the date of the post
3. Open the page in the Wagtail admin: http://localhost:8000/admin/pages/1327/edit/
4. Delete all text ("date filed") in the "Date stamp descriptor" field
5. Publish the page
6. See that each post has no date description

## Screenshots

Screen Shot 2019-09-27 at 10 06 44 AM
Screen Shot 2019-09-27 at 10 20 44 AM

## Checklist

- [x] PR has an informative and human-readable title
- [x] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [x] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [x] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated
- [x] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### CODE REVIEW

**Comment by schbetsy** [comment]
@cwdavies Thanks for reviewing. For normal python code, I'd say your feedback is spot on. However, migrations files are machine generated, and aren't really intended to be modified or read by people. [Here's the Django documentation if you want to read more](https://docs.djangoproject.com/en/1.11/topics/migrations/).

File: cfgov/v1/atomic_elements/organisms.py [hunk 1]
```diff
@@ -854,7 +854,11 @@ class FilterableList(BaseExpandable):
         required=False,
         help_text='Additional explanation for the notification that '
                   'will be displayed if there are no posts to filter.')
-    post_date_description = blocks.CharBlock(default='Published')
+    post_date_description = blocks.CharBlock(
+        required=False, label='Date stamp descriptor',
```
Comment (Scotchester) [inline cfgov/v1/atomic_elements/organisms.py]: Trivial: There are a lot of older places where we're not consistent about this, but I think our preference is, when arguments must be broken onto multiple lines, to begin each argument on its own line.

File: cfgov/v1/atomic_elements/organisms.py [hunk 2]
```diff
@@ -854,7 +854,11 @@ class FilterableList(BaseExpandable):
...
+        help_text='Strongly encouraged to help users understand the '
+        'action that the date of the post is linked to, i.e. published, '
```
Comment (Scotchester) [inline cfgov/v1/atomic_elements/organisms.py]: Trivial: I think we also prefer to align the opening quote marks of subsequent lines of a long string with the first line.

File: cfgov/v1/atomic_elements/organisms.py (hunk 1 above)
Comment (schbetsy) [reply cfgov/v1/atomic_elements/organisms.py]: Sounds good 👍

File: cfgov/v1/atomic_elements/organisms.py (hunk 2 above)
Comment (schbetsy) [reply cfgov/v1/atomic_elements/organisms.py]: Will do 👍

**Comment by schbetsy** [comment]
@cwdavies If you have any other concerns or questions, feel free to ask. If not, could you update your review to approve, which will allow me to merge?

**Pull request merged**